scrapy/extensions/feedexport.py

```diff
@@ -277,18 +277,21 @@ def _store_in_thread(self, file):
 class FeedSlot:
     def __init__(
         self,
-        file,
-        exporter,
```

Comment:
This now changes a public class, as we made FeedSlot public recently. @kmike @Gallaecio thoughts?

Reply:
The only backward-compatible option I can think of at the moment is to keep the current class unchanged, deprecate it, and create a new class with the new API with a new class name (`BatchSlot`?).

Reply:
We decided that as this only changes the `__init__` signature and the current usage of this class is to receive it in the `feed_slot_closed` signal handler, it's fine to just change this.